translations/strings.yaml

```diff
@@ -671,6 +671,24 @@ subscriptionManagement:
   planValueYearly:
     value: "%1 Yearly"
     comment: The %1 represents a localized currency string.
+  planValueMonthlyPlusTax:
+    value: "%1 Monthly + tax"
```

Comment:
One thing I'm missing: why "X Monthly" but also "X/month"? Is one of the 2 obsolete, is this a testing? I expect most locales to just translate them the same way.

Reply:
We use different verbiage in different parts of the app

<details>
<summary>Images</summary>

![IMG_54B1CC17174D-1](https://user-images.githubusercontent.com/15353801/220666872-5c7bd1aa-20d3-478b-bb31-187800fa5acd.jpeg)

<img width="248" alt="Screen Shot 2023-02-22 at 10 18 14 AM" src="https://user-images.githubusercontent.com/15353801/220666903-243639d5-0ab4-4925-917b-c8a03bd878f1.png">

</details>